portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,6 +14,12 @@
 
 package com.google.sps.servlets;
 
+import com.google.appengine.api.datastore.*;
+<<<<<<< HEAD
+import com.google.gson.*;
+=======
+import com.google.gson.Gson;
+>>>>>>> 938c2c3e12f97becd762dd1b5123b76f4a042617
```

Comment:
This looks like a remnant from a merge conflict
```suggestion
import com.google.gson.Gson;
```